pages/linux/arpspoof.md

```diff
@@ -0,0 +1,14 @@
+# arpspoof
+
+> Intercept packets on a switched LAN.
+> Redirects packets from a target host (or all hosts) on the LAN intended for another host on the LAN by forging ARP replies.
+> More information: <https://manned.org/arpspoof>.
+
+- Make a simple man-in-the-middle attack between two local hosts (run both at the same time, via `screen` or multiple terminal sessions):
+
+`sudo arpspoof -i {{eth0}} -t {{192.168.0.1}} {{192.168.0.2}}`
+`sudo arpspoof -i {{eth0}} -t {{192.168.0.2}} {{192.168.0.1}}`
```

Comment:
```suggestion
```

Code examples are one-liners.

Reply:
This needs two lines though to work correctly, and in this case I think it's justified, as it's one of the main purposes of the program - to test against MITM attack possibilities (in environments where you have explicit approval).
I think it's better for it to be two lines rather than one line with `;`.

Reply:
TlDr pages should have one-line examples. Linter doesn't accept (by default) multiline code examples.

Reply:
Yeah, unfortunately we can't have more than 1 line in each example because of a limitation in our linter (which would cause all subsequent pull requests to fail the linter if we merge 1 PR that doesn't pass the linter).